packages/itwinui-react/src/core/DatePicker/DatePicker.tsx

```diff
@@ -189,6 +189,38 @@ type DatePickerProps = {
    * @default false
    */
   showYearSelection?: boolean;
+  /**
+   * Allows props to be passed for calendar month year
+   */
+  calendarProps?: React.ComponentProps<'div'>;
+  /**
+   * Allows props to be passed for label inside polite
+   */
+  politeLabelProps?: React.ComponentProps<'span'>;
```

Comment:
Not sure this is needed? Did you read about `aria-live='polite'`?

Reply:
Yes I just read the MDN docs. I put it in originally because I didn't know if it was something the user may want to customize, but I can see now it may not have much of a function. I'll comment it out for now

Reply:
Remove it.